tsdate/core.py

```diff
@@ -932,7 +932,7 @@ def constrain_ages_topo(ts, post_mn, eps, nodes_to_date=None, progress=False):
 def date(
     tree_sequence,
     mutation_rate,
-    Ne=None,
```

Comment:
It would be good to retain `Ne` as a (deprecated) alias for population_size, for backwards compatibility. We can raise a warning that this is deprecated, if you like.